lib/execute.ex

```diff
@@ -1,12 +1,18 @@
 defmodule Execute do
-  def run_module(module) do
+  def run_module(module, callback \\ fn(result, koan) -> nil end) do
```

Comment:
Is this default argument for tests? Would it look worse if we passed a dummy function in tests instead of making it the default on production side?